pkg/utils/utils.go

```
@@ -229,14 +229,14 @@ func (u *utils) IsSNOCluster() (bool, error) {
 	return infra.Status.ControlPlaneTopology == configv1.SingleReplicaTopologyMode, nil
 }
 
-func (u *utils) IsBMInfra() (bool, error) {
+func (u *utils) GetPlatformType() (configv1.PlatformType, error) {
```

Maybe we should return an error if the platform type is not supported (not none, bm or aws)?

---

There other platforms are valid but they are not supported, so if someone will try to run the test we another platform type it will fail as it will assume to be aws.
This is why returning an error saying the platform is **not supported** in our application is a good solution in my opinion.

---

i added it at the end since on our doc we are just talking about that supported cases if we will support more on the future so we need to update the slice